datadogclient/datadog_client_test.go

```diff
@@ -134,7 +134,7 @@ var _ = Describe("DatadogClient", func() {
 		var payload datadogclient.Payload
 		err = json.Unmarshal(bodies[0], &payload)
 		Expect(err).NotTo(HaveOccurred())
-		Expect(payload.Series).To(HaveLen(4))
+		Expect(payload.Series).To(HaveLen(5))
```

Comment:
It threw me off for a moment that this didn't just double, but then I remembered that it has 3 metrics that are outside of the normal metric pipeline